modules/uncertainty/R/get.parameter.samples.R

```diff
@@ -202,7 +206,7 @@ get.parameter.samples <- function(settings,
                                                env.samples, ens.sample.method, param.names)
     }
   }
-  
+  #ensemble.samples <- NULL
```

Comment:
why add this commented out line?

Reply:
Sorry, it was part of a previous attempt. It should be deleted, but I forgot.